pareto/strategic_water_management/strategic_produced_water_optimization.py

```diff
@@ -4746,8 +4857,18 @@ def PumpCostRule(b, l1, l2):
         the cost of pumps.
         """
         # In the co_optimize method the objective should include the cost of pumps. So,
-        # delete the original objective to add a modified objective in this method.
-        del model.objective
+        # deactivate the original objectives to add a modified objective in this method.
+        model.objective_Cost.deactivate()
```

Comment:
I believe we never activate the objective after deactivating them in 4711 - 4717. This part of the code may not be needed.

Reply:
@JavalVyas2000 the lines you are referring to are in different sections of an `if`/`elif` block - lines 4711-4717 corresponds to the config option for `Hydraulics.co_optimize` and 4861-4866 is in the section for `Hydraulics.co_optimize_linearized`. So it is necessary in both places.